crates/atuin/src/command/client/scripts.rs

```diff
@@ -0,0 +1,573 @@
+use std::collections::HashMap;
+use std::collections::HashSet;
+use std::path::PathBuf;
+
+use atuin_scripts::execution::template_script;
+use atuin_scripts::{
+    execution::{build_executable_script, execute_script_interactive, template_variables},
+    store::{ScriptStore, script::Script},
+};
+use clap::{Parser, Subcommand};
+use eyre::{Result, bail};
+use tempfile::NamedTempFile;
+
+use atuin_client::{database::Database, record::sqlite_store::SqliteStore, settings::Settings};
+use tracing::debug;
+
+#[derive(Parser, Debug)]
+pub struct NewScript {
+    #[arg(short, long)]
+    pub name: String,
```

Comment:
Thoughts on allowing this to be positional? The other commands that take a script name do so, and it's a required parameter.

Reply:
yep definitely, that's much better